### PR TITLE
Update to Gradle 4.0.2

### PR DESCRIPTION
**Pull request by AustinShalit**

### CODE REVIEW

File: src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java [hunk 1]
```diff
@@ -38,7 +39,7 @@
    * Compares tree items alphabetically.
    */
   private static final Comparator<TreeItem<NetworkTableEntry>> alphabetical
-      = Comparator.comparing(item -> item.getValue().getKey().toLowerCase());
+      = Comparator.comparing(item -> item.getValue().getKey().toLowerCase(Locale.getDefault()));
```
Comment (SamCarlberg) [inline src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java]: Not relevant to this PR

Comment (AustinShalit) [inline src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java]: Yes it is.  See the Gradle release notes:

https://docs.gradle.org/current/release-notes.html#version-of-pmd-has-been-upgraded

Comment (AustinShalit) [inline src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java]: (PMD should have caught this before but there was a bug)

Comment (SamCarlberg) [inline src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java]: I stand corrected

**Comment by JLLeitschuh** [comment]
Is this not passing because of #70 ?

**Comment by SamCarlberg** [comment]
```
edu.wpi.first.shuffleboard.components.NetworkTableTreeTest > testSort FAILED
    java.lang.ClassCastException: com.sun.javafx.scene.control.skin.LabeledText cannot be cast to javafx.scene.control.TreeTableCell
        at edu.wpi.first.shuffleboard.components.NetworkTableTreeTest.assertCellIndex(NetworkTableTreeTest.java:129)
        at edu.wpi.first.shuffleboard.components.NetworkTableTreeTest.testSort(NetworkTableTreeTest.java:121)
```

Well, kinda. Randomly failing UI tests, definitely.

**Comment by JLLeitschuh** [comment]
Sounds like you shouldn't be making casts that are illegal.

**Comment by SamCarlberg** [comment]
It's a test framework issue. TestFX found the wrong item on a lookup. Rerunning the build will most likely make it go away

File: src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java (hunk 1 above)
Comment (nightpool) [inline src/main/java/edu/wpi/first/shuffleboard/widget/NetworkTableTreeWidget.java]: Using Locale.getDefault() here is exactly identical to existing behavior. Why make it explicit?

**Comment by SamCarlberg** [comment]
@AustinShalit please rebase or merge master

**Comment by SamCarlberg** [comment]
Closed by #73

**Pull request closed without merging**